index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The AudioWorklet object allows importing user-supplied Javascript code
+          that runs on the audio rendering thread. This processing mechanism
+          ensures the aligned execution of the script code with other built-in
+          <a>AudioNode</a>s in the audio graph. The AudioWorklet object imports
+          and stores the definition of custom audio processor and then an
+          <a>AudioWorkletNode</a> object can be constructed based on the
+          processor definition. Note that the registered definition in an
+          AudioWorklet is valid for the entire global scope, therefore any
+          <a>AudioContext</a> in the same global scope can be used to create the
+          custom node.
+        </p>
+        <p>
+          An AudioWorklet object provides the capability to import module
+          scripts into its associated <a>AudioWorkletGlobalScope</a>. The user
+          agent can then create <a>AudioWorkletProcessor</a> classes registered
+          on the <a>AudioWorkletGlobalScopes</a> and construct their instances.
+          A processor object represents an audio processing module that runs on
+          the audio <a>rendering thread</a>. After the processor definition is
+          successfully stored, an instance of <a>AudioWorkletNode</a> can be
+          created in the main global scope. This is the counterpart of the
+          processor but lives on the main <a>control thread</a>. These two
+          components can communicate via message passing.
+        </p>
+        <dl title="interface AudioWorklet : Worklet" class="idl">
+          <dt>
+            Promise&lt;void&gt; import(DOMString moduleUrl)
+          </dt>
+          <dd>
+            <p>
+              Imports a module script returns a promise once it is fully loaded.
```

Comment:
"script returns" -> "script and returns"
"promise once it is fully loaded" should be rephrased.  Maybe returns a promise that resolves when fully loaded?